dfastmi/gui/dialog_model.py

```diff
@@ -122,17 +122,21 @@ def _get_condition_configuration(self, config : ConfigParser, reach : AReach, re
         num_files = min(len(reference_files), len(measure_files))
         
         # loop over conditions cond = "C1", "C2", ...        
-        for i, discharge in enumerate(reach.hydro_q[:num_files]): # Ensure it loops until the minimum length
-            cond = f"C{i+1}"
-            config.add_section(cond)            
-            config[cond]["Discharge"] = str(discharge)
-             # Check if the index is within the bounds of the lists
-            if i < len(reference_files):
-                config[cond]["Reference"] = reference_files[i]
-            else:
-                config[cond]["Reference"] = ""  # Default value if index is out of range
-
-            if i < len(measure_files):
-                config[cond]["WithMeasure"] = measure_files[i]
-            else:
-                config[cond]["WithMeasure"] = ""  # Default value if index is out of range
+        i = 0
+        for discharge in enumerate(reach.hydro_q[:num_files]): # Ensure it loops until the minimum length
```

Comment:
Maybe move this to a descriptive method?
E.g.
set_all_discharges_references_measures